src/xtgeo/grid3d/grid.py

```diff
@@ -1310,6 +1312,42 @@ def get_dz(self, name="dZ", flip=True, asmasked=True, mask=None):
 
         return deltaz
 
+    def get_dx(self, name="dX", asmasked=True):
+        """Return the dX and dY as GridProperty object.
+
+        The values lengths are projected to a constant Z.
+
+        Args:
+            name (str): names of properties
+            asmasked (bool). If True, make a np.ma array where inactive cells
+                are masked.
+
+        Returns:
+            XTGeo GridProperty objects containing dx.
+        """
+        return _grid_etc1.get_dx(self, name=name, asmasked=asmasked)
+
+    def get_dy(self, name="dY", asmasked=True):
+        """Return the dX and dY as GridProperty object.
+
+        The values lengths are projected to a constant Z.
+
+        Args:
+            name (str): names of properties
+            asmasked (bool). If True, make a np.ma array where inactive cells
+                are masked.
+
+        Returns:
+            Two XTGeo GridProperty objects (dx, dy).
+        """
+        return _grid_etc1.get_dy(self, name=name, asmasked=asmasked)
+
+    @deprecation.deprecated(
+        deprecated_in="2.16",
+        removed_in="4.0",
+        current_version=xtgeo.version,
+        details="Use xtgeo.Grid.get_dx() and/or xtgeo.Grid.get_dy() instead.",
+    )
     def get_dxdy(self, names=("dX", "dY"), asmasked=False):
```

Comment:
Is it necessary to deprecate this?

Reply:
I think it will be cleaner to deprecate this as `grid.get_dxdy() == grid.get_dx(), grid.get_dy()`.

Reply:
I see too many deprecations give a lot of screen noise, so I would like to keep it for backward compatibility

Reply:
We could set the deprecation for a later version if that is preferable?

Reply:
Yes we could do that to allow for some considerations